notebooks/official/ml_metadata/vertex-pipelines-ml-metadata.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>This should be moved to below creating the BUCKET_URI. Use latest notebook template</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/826/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>